libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -185,64 +178,64 @@ irods::error start(irods::default_re_ctx& _u,const std::string& _instance_name)
 }
 
 irods::error stop(irods::default_re_ctx& _u,const std::string& _instance_name) {
+    (void) _u;
```

Comment:
If `_u` is not used, you can just remove the variable name. I'm guessing there was a copy-paste from `start` here...

Reply:
So just remove the cast and leave the parameter?

Reply:
He's referring to the `_u` parameter in the function signature.

In C++, you don't have to provide a name for unused parameters. For example, the function should have the following signature:
```cpp
irods::error stop(irods::default_re_ctx&, const std::string& _instance_name) {
```